understanding/pointer-gestures.html

```diff
@@ -35,7 +35,7 @@ <h1>達成基準 2.5.1: ポインタのジェスチャを理解する</h1>
       <main>
          <section id="intent">
             <h2>意図</h2>
-            <p>The intent of this Success Criterion is to ensure that content can be operated using simple inputs on a wide range of pointing devices. This is important for users who cannot perform complex gestures, such as multi-point or path-based gestures, in a precise manner, either because they may lack the accuracy necessary to carry them out or because they use a pointing method that lacks the capability or accuracy.</p>
+            <p>この達成基準の意図は、広範囲なポインティング デバイスの単純な入力でコンテンツを操作できるようにすることである。これは、ユーザーが操作に必要な正確さを欠いてたり、機能や正確さを欠くポインティング方法を利用することにより、マルチポイントや軌跡に基づくジェスチャーのような、複雑なジェスチャーを正確に実行できないユーザーにとって重要である。</p>
```

Comment:
### 主に1文目
- ensure tnatが訳出できていないような気が https://ejje.weblio.jp/content/ensure
- WCAG 2.1では「ポインティングデバイス」としているのでスペースなしで。 c.f. https://waic.github.io/wcag21/guidelines/#dfn-assistive-technologies
- ユーザー→利用者（翻訳ガイドラインはまだ整備されていませんが）

### 主に2文目
- either because... or because～ の訳ができていないような気が。
- in a precise mannerがどこに行ってしまったのか… ニュアンスとしては、「複雑なジェスチャー (<b>正確には</b>、マルチポイントや軌跡に基づくジェスチャーのような)」みたいな感じだと思います。
- pointing method ここは「ポインティング方法」より「ポインティング方式」の方が自然でしょうか？ https://ejje.weblio.jp/content/method